ISHPermissionKit.xcodeproj/project.pbxproj

```diff
@@ -7,56 +7,30 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		1B06A05A1D70949A00C7F8ED /* ISHPermissionKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = DC6A46B119D712AC00B11E88 /* ISHPermissionKit.framework */; };
-		1B06A05B1D70949A00C7F8ED /* ISHPermissionKit.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = DC6A46B119D712AC00B11E88 /* ISHPermissionKit.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		1B09324A1D7069C70095A74A /* ISHPermissionRequestContacts.m in Sources */ = {isa = PBXBuildFile; fileRef = 1B0932481D7069C70095A74A /* ISHPermissionRequestContacts.m */; };
 		1B09324C1D7069D90095A74A /* ISHPermissionRequestContacts.h in Headers */ = {isa = PBXBuildFile; fileRef = 1B09324B1D7069D90095A74A /* ISHPermissionRequestContacts.h */; };
 		1B09324D1D7069D90095A74A /* ISHPermissionRequestContacts.h in Headers */ = {isa = PBXBuildFile; fileRef = 1B09324B1D7069D90095A74A /* ISHPermissionRequestContacts.h */; };
-		1B09324E1D7069D90095A74A /* ISHPermissionRequestContacts.h in Headers */ = {isa = PBXBuildFile; fileRef = 1B09324B1D7069D90095A74A /* ISHPermissionRequestContacts.h */; };
```

Comment:
These files have not been removed. I assume these deletions stem from removing the test app.